linker/spm.jp0.lst

```diff
@@ -3,6 +3,9 @@
 800030c8:firstRel
 800030cc:lastRel
 
+
+803F0E88:superDimentioHeadDefenses
+803F0EA0:superDimentioBodyDefenses
```

Comment:
These should be grouped in with the rest of the hard mode symbols (same in us0, us2)